articles/azure-app-configuration/push-kv-devops-pipeline.md

```diff
@@ -41,6 +41,22 @@ Now that your service connection is created, find the name of the service princi
 1. Note the **Display name** listed.
 ![Screenshot shows the service principal display name.](./media/service-principal-display-name.png)
 
+## Create a service connection (manual setup)
+
+1. In Azure DevOps, go to the project containing your target pipeline and open the **Project settings** at the bottom left.
+1. Under **Pipelines** select **Service connections** and select **New service connection** in the top right.
+1. Select **Azure Resource Manager**.
+1. In the **Authentication method** dialog, select **Service principal (manual)** and enter the Service principal details.
+1. Select the Environment name (such as Azure Cloud, Azure Stack, or an Azure Government Cloud).
+1. Enter the information about your service principal into the Azure subscription dialog textboxes.
+     - Subcription Id
+     - Subscription Name 
+     - Service principal Id 
+     - Enter the service principal client key (Certificate is currently unsupported for the App Configuration task)
+     - Tenant Id
+1. Choose **Verify connection** to validate settings you entered.
+1. Enter a user-friendly **Connection name** to use when referring to this service connection. 
+
```

Comment:
I don't think we need to add a whole new section. All we need is to add a link to the other doc. Suggest changing step 4 to:

4. In the Authentication method dialog, select \*\*Service principal (automatic)\*\* to create a new service principal or select \*\*Service principal (manual)\*\* to \[use an existing service principal\]\(azure/devops/pipelines/library/connect-to-azure?view=azure-devops#use-spn\).

We also need to make a minor update of the following paragraph:

- If you created a new service principal, ~Now that your service connection is created,~ find the name of the service principal assigned to ~it~ *the service connection*.